sources/background_scripts/image_context_menu.js

```diff
@@ -109,4 +121,13 @@
         });
         update_context_menu_item_visibility();
     });
+    
+    define(["background_scripts/notifications"],
```

Comment:
I have not tested this as I am on my way to bed, but most likely this line here is the problem. `"background_scripts/notification"` should be added to the array at the original `define`, as well as the `notification_module` as an additional parameter next to `configuration_module`. There should be only one `define` call in total.

Reply:
I apologize if something I said mislead you with my wording. [Here ](http://www.requirejs.org/docs/api.html#defdep) is the relevant documentation.